nodes/standalone/Cargo.toml

```
@@ -41,6 +41,7 @@ sc-executor = {workspace = true}
 sc-consensus-grandpa.workspace = true
 sc-keystore.workspace = true
 sc-service = {workspace = true}
```

That's a little out of scope for this PR but could you include this change:

```suggestion
sc-service.workspace = true
```

---

Updated in: [b41b3bf](https://github.com/KILTprotocol/kilt-node/pull/541/commits/b41b3bf800e3d925f5f2ae94ce85359a34f78bd1)